internal/terraform/testdata/plan-self-ref-multi-all/main.tf

```diff
@@ -1,4 +1,4 @@
 resource "aws_instance" "web" {
-    foo = "${aws_instance.web.*.foo}"
```

Comment:
improved evaluation found a very old invalid test which was assigning a `list(list(string))` to a `list(string)` (though that could have been flattened in the legacy type system, but was never caught post-0.12)